docs/narr/security.rst

```diff
@@ -290,6 +290,11 @@ properties of the instance.
        def __init__(self, owner):
            self.owner = owner
 
+**warning:** writing __acl__ as properties is discouraged, as AttributeErrors
```

Comment:
Use .rst block and wrap appropriately:

```rst
.. warning::
```

Use double-backticks for inline code, e.g.:

```rst
``__acl__``
```